server/api/src/rs_get_resource_info_for_operation.cpp

```diff
@@ -1,34 +1,39 @@
-#include "irods/get_resource_info_for_operation.h"
+#include "irods/rs_get_resource_info_for_operation.hpp"
 
 #include "irods/getRemoteZoneResc.h"
+#include "irods/get_resource_info_for_operation.h"
+#include "irods/irods_error.hpp"
 #include "irods/irods_logger.hpp"
 #include "irods/irods_resource_backport.hpp"
 #include "irods/irods_resource_redirect.hpp"
+#include "irods/rcMisc.h"
+#include "irods/rodsConnect.h"
 #include "irods/rodsError.h"
+#include "irods/rodsErrorTable.h"
 
-#include <nlohmann/json.hpp>
 #include <fmt/format.h>
+#include <nlohmann/json.hpp>
 
+#include <algorithm>
 #include <array>
 #include <cstring> // for strdup
 #include <string>
 
-namespace
+int rs_get_resource_info_for_operation(RsComm* _comm, DataObjInp* _dataObjInp, char** _out_info)
```

Comment:
Might get dinged for trailing return type here

Reply:
Oh right. Will wait for the clang-tidy report before changing.

Reply:
Clang-tidy came back happy.